src/index.js

```diff
@@ -14,6 +15,7 @@ program
   .option('-p, --hydra-prefix [hydraPrefix]', 'The hydra prefix used by the API', 'hydra:')
   .option('-g, --generator [generator]', 'The generator to use, one of "react", "react-native", "vue", "admin-on-rest"', 'react')
   .option('-t, --template-directory [templateDirectory]', 'The templates directory base to use. Final directory will be ${templateDirectory}/${generator}', `${__dirname}/../templates/`)
+  .option('-s, --resource-type [resourceType]', 'Hydra or Swagger', 'hydra')
```

Comment:
May I suggest `-f --format` instead?

Should be ` .option('-f, --format [hydra|swagger]', '"hydra" or "swagger', 'hydra')`